### PR TITLE
Redesigned landing page

### PR DESCRIPTION
**Pull request by jrubenoff**
screen shot 2016-04-16 at 9 37 30 am

Tested on Chrome, Safari, FF, IE9

### CODE REVIEW

**Comment by ajb** [comment]
Looks great! One bug I found:

img

Also, can you make sure that `index.html` _doesn't_ get committed to this branch?

File: Rakefile [hunk 1]
```diff
@@ -71,52 +196,146 @@ task :generate_preview do
...
+                font-size: 150%;
+              }
             }
           }.squish.html_safe
         }
 
         body {
+
+          div(class: 'header') {
+            a(href: 'http://github.com/dobtco/palat',
```
Comment (ajb) [inline Rakefile]: Should be `https:`

File: Rakefile [hunk 2]
```diff
@@ -71,52 +196,146 @@ task :generate_preview do
...
+              title: 'Fork Palat on Github',
+              class: 'logo') {
+              img(src: 'http://i.imgur.com/4JfuQgs.png',
```
Comment (ajb) [inline Rakefile]: Let's get this image into the repo.

File: Rakefile [hunk 3]
```diff
@@ -71,52 +196,146 @@ task :generate_preview do
...
+          }
+
+          div(class: 'keycodes_wrapper') {
+            div(class: 'keycode keycode_tilde') {
+              strong '~'
+              span 'Toggle variables'
+            }
+
+            div(class: 'keycode keycode_f') {
+              strong 'f'
```
Comment (ajb) [inline Rakefile]: The "f" makes it look like a Facebook logo. How about "e"?

File: Rakefile [hunk 4]
```diff
@@ -71,52 +196,146 @@ task :generate_preview do
...
+              span 'Toggle error state'
+            }
+          }
+
           test_color_array.each do |color|
             render_section(color)
           end
+
+          div(class: 'footer') {
+            p "Made by <a href='http://dobt.co'>The Department of Better Technology.</a>".html_safe
```
Comment (ajb) [inline Rakefile]: `https://www.dobt.co/`

File: Rakefile [hunk 5]
```diff
@@ -71,52 +196,146 @@ task :generate_preview do
...
+            p { a 'Fork Palat on Github', href: 'http://www.github.com/dobtco/palat/' }
```
Comment (ajb) [inline Rakefile]: `https`

**Comment by jrubenoff** [comment]
Just addressed all your comments.

File: Rakefile [hunk 6]
```diff
@@ -71,52 +196,146 @@ task :generate_preview do
...
+              span 'Toggle error state'
+            }
+          }
+
           test_color_array.each do |color|
             render_section(color)
           end
+
+          div(class: 'footer') {
+            p "Made by <a href='https://dobt.co'>The Department of Better Technology.</a>".html_safe
```
Comment (ajb) [inline Rakefile]: Gotta have `www`

**Comment by ajb** [comment]
✨ 

Looks fantastic. Let's think about adding an "open source" section to our splash page soon. (https://github.com/dobtco/dobt/issues/1069)

**Pull request merged**